### PR TITLE
CSS changes for user report

### PR DESCRIPTION
**Pull request by mikehelmick**
image

**Release Note**

```release-note
CSS Tweaks for user report webview.
```

### CODE REVIEW

**Review by whaught — APPROVED** [review]

**Comment by allenx** [comment]
1. The alert has border-color, can we remove the border or make it transparent? 
2. It looks like I have some bad calculations on the continue button's corner radius. Can we make it a fixed value which is not based on screen size. The number looked good on my small screen phone, I didn't know it would look this much rounder on bigger screens.

**Comment by mikehelmick** [comment]
I removed the border on the alert and made the button have the same fixed radius as the other elements, `20px`

File: assets/enx-redirect/report/header.html [hunk 1]
```diff
@@ -40,6 +52,23 @@
...
+.continue-btn {
+  display: inline-block;
+  font-weight: 600;
+  line-height: 1.5;
+  color: #ffffff;
+  text-align: center;
+  text-decoration: none;
+  vertical-align: middle;
+  cursor: pointer;
+  -webkit-user-select: none;
```
Comment (sethvargo) [inline assets/enx-redirect/report/header.html:65]: You can add a class of `user-select-none` to the element instead of doing this.

**Review by sethvargo — COMMENTED** [review]
> The number looked good on my small screen phone, I didn't know it would look this much rounder on bigger screens.

Given this will only ever appear on phones via a POST request in a webview, I think that's okay?

**Pull request merged**